kratos/tests/sources/test_properties.cpp

```diff
@@ -37,7 +37,7 @@ namespace Kratos {
 			Properties property(0);
 			KRATOS_CHECK_IS_FALSE(property.HasVariables());
 
-			Variable<double> variable("TEST");
+			Variable<double> variable(TEMPERATURE);
 			property.SetValue(variable, 1.0);
```

Comment:
Minor comment: you don't need to create a copy of the temperature here, you could just call `property.SetValue(TEMPERATURE,1.0);`.